mysql-test/suite/binlog_gtid/t/percona_binlog_skip_flush_commands.test

```diff
@@ -0,0 +1,204 @@
+--source include/have_log_bin.inc
```

Comment:
Ideally there could be a binlog_nogtid test too that drops the check for gtids, but I am not sure whether effort/payoff ratio is good enough for that